mkmsdk/api_map.py

```diff
@@ -1,4 +1,3 @@
-# flake8: noqa
```

Comment:
Given the nature of this file I prefer if we keep this flag so flake8 will skip it, otherwise it will show lots of errors.

Reply:
Interesting, I didn't even know this line was there and I certainly didn't mean to remove it. I'll add it back.